server/util/statusz/statusz.go

```diff
@@ -0,0 +1,188 @@
+package statusz
+
+import (
+	"context"
+	"html/template"
+	"net/http"
+	"os"
+	"os/user"
+	"path/filepath"
+	"sort"
+	"sync"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/server/version"
+)
+
+const templateContents = `<!DOCTYPE html>
+<html>
+  <head>
+    <title>/statusz</title>
+    <style>
+	body {
+	  font-family: sans-serif;
+	  background: #fff;
+	}
+	h1 {
+	  clear: both;
+	  width: 100%;
+	  text-align: center;
+	  font-size: 120%;
+	  background: #eeeeff;
+	}
+	h2 {
+	  clear: both;
+	  width: 100%;
+	  font-size: 110%;
+	  text-align: center;
+	  background: #fffddd;
+	}
+	.lefthand {
+	  float: left;
+	  width: 80%;
+	}
+	.righthand {
+	  text-align: right;
+	}
+    </style>
+  </head>
+    <h1>Status for {{.BinaryName}}</h1>
+  <div>
```

Comment:
as mentioned above, i'd apply a class to this div and (once it's closed - see below comment) apply `display: flex` to it